README.md

```diff
@@ -179,30 +174,29 @@ Once CMake is configured, building the library is as simple as typing 'make'.
 This will build all library components as well as 'mlpack_test'.
 
     $ make
-
-You can specify individual components which you want to build, if you do not
-want to build everything in the library:
+    
+If you do not want to build everything in the library, individual components of the build can be specified:
 
     $ make mlpack_pca mlpack_knn mlpack_kfn
 
 If the build fails and you cannot figure out why, register an account on Github
-and submit an issue; the mlpack developers will quickly help you figure it out:
+and submit an issue. The mlpack developers will quickly help you figure it out:
 
 [mlpack on Github](https://www.github.com/mlpack/mlpack/)
 
 Alternately, mlpack help can be found in IRC at `#mlpack` on irc.freenode.net.
 
-If you wish to install mlpack to `/usr/local/include/mlpack/` and `/usr/local/lib/`
-and `/usr/local/bin/`, once it has built, make sure you have root privileges (or
-write permissions to those three directories), and simply type
+If you wish to install mlpack to `/usr/local/include/mlpack/` , `/usr/local/lib/`
+and `/usr/local/bin/`, make sure you have root privileges (or write permissions to those three directories) once it has built, 
```

Comment:
Actually the ordering of the `once it has built` is important, but honestly I think we can drop that phrase since `make install` will build the library if it hasn't already been built anyway.